go

```diff
@@ -58,6 +58,12 @@ def init
   exec_cmd 'bundle install'
 end
 
+def update_components
+  exec_cmd 'npm install -g bower bower-installer'
```

Comment:
I was debating it, but this keeps the barrier to entry low. Since the dependencies are being checked in, Node/Bower isn't actually needed unless installing new packages or updating existing ones.